Automattic/jetpack
# A4A Plugin: Add ConnectedCard component for site connected state

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/128

## Proposed changes:

* Adds a card for the "connected" site state.
  * Shows a success message by default, with a link to the A4A dashboard.
  * Includes a disconnect button below the card, which allows the user to remove their site connection.
  * After disconnecting, a custom notice is shown to confirm the success, and allows the user to re-connect their site.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/automattic-for-agencies-dev/issues/128

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Visit `/wp-admin/options-general.php?page=automattic-for-agencies-client`
* Cycle through a full unconnected -> connected -> unconnected -> connected cycle to validate all flows. See screenshots for reference.

## Screenshots
<img width="1125" alt="Screenshot 2024-04-04 at 5 53 44 PM" src="https://github.com/Automattic/jetpack/assets/10933065/cfe9f74c-079d-4302-87b3-8af69c26ced9">
<img width="1118" alt="Screenshot 2024-04-04 at 5 53 53 PM" src="https://github.com/Automattic/jetpack/assets/10933065/463872e0-074c-4186-9a76-6b1f0b5806e7">
<img width="1119" alt="Screenshot 2024-04-04 at 5 53 59 PM" src="https://github.com/Automattic/jetpack/assets/10933065/82548e53-9c42-4bad-9b9e-847026e8be9a">
<img width="1118" alt="Screenshot 2024-04-04 at 5 54 23 PM" src="https://github.com/Automattic/jetpack/assets/10933065/c5c554ca-4797-449b-bb8d-91f497be8b71">
